src/components/Navbar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { Route, Routes, Link } from 'react-router-dom';
+import logo from '../images/planet.png';
+import '../styles/Navbar.css';
+import RocketsList from './navigation/RocketsList';
+import MissionsList from './navigation/MissionsList';
+import DragonsList from './navigation/DragonsList';
+import MyProfile from './navigation/MyProfile';
+
+function Navbar() {
+  const [activeLink, setActiveLink] = useState('/');
+
+  useEffect(() => {
+    setActiveLink('/rockets'); // Set '/rockets' as the default active link
+  }, []);
+
+  return (
+    <>
+      <nav className="navbar">
+        <ul>
+          <li>
+            <Link
+              to="/"
+              onClick={() => setActiveLink('/rockets')}
+              className={activeLink === '/rockets' ? 'active' : ''}
+            >
+              <img src={logo} alt="logo" />
+            </Link>
```

Comment:
- [ ] Please use `<NavLink />` for the page navigation links and style active class instead of `Link`. The` NavLink` is used when you want to highlight a link as active. So, on every routing to a page, the link is highlighted according to the activeClassName . `Link `is for links that need no highlighting. And `a` is for external links.